pkg/storage/postgres/postgres.go

```diff
@@ -81,6 +82,7 @@ func New(uri string, cfg *common.Config) (*Postgres, error) {
 		logger:                 cfg.Logger,
 		maxTuplesPerWriteField: cfg.MaxTuplesPerWriteField,
 		maxTypesPerModelField:  cfg.MaxTypesPerModelField,
+		currentTime:            storage.DefaultTimeProvider{},
```

Comment:
```suggestion
		timeProvider:            storage.DefaultTimeProvider{},
```

Reply:
Obsolete as we don't have the time provider anymore